old-source/transform.R

```diff
@@ -0,0 +1,65 @@
+# from the babeldown package
```

Comment:
this script is how I created the Quarto book. I also had to make a few manual tweaks in the Rmd that ends up in `what_r_does_wrong.qmd` (white space) for the conversion to go well.